src/patternfly/components/Card/card.scss

```diff
@@ -344,6 +344,11 @@
     --#{$card}__actions--MarginBlockStart: 0;
     --#{$card}__actions--MarginBlockEnd: 0;
   }
+
+  &:has(button, a) {
+    padding: 0;
+    margin: 0;
+  }
```

Comment:
If I'm reading the issue correctly, this should be scoped just to the pf-m-clickable card. As is, this is removing padding and margin for any card actions that contain a link or button - which is probably all or most of them, right?